src/contracts/pods/EigenPod.sol

```diff
@@ -761,6 +775,13 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         return (int256(newAmountWei) - int256(currentAmountWei));
     }
 
+    // reference: https://github.com/ethereum/consensus-specs/blob/ce240ca795e257fc83059c4adfd591328c7a7f21/specs/bellatrix/beacon-chain.md#compute_timestamp_at_slot
+    function _computeSlotAtTimestamp(uint64 timestamp) internal view returns (uint64) {
```

Comment:
fine with this, but does it make snese to put this in the beacon chain lib and rename to BeaconChainUtils.sol?

cc @ChaoticWalrus

Reply:
hmm, it would need to take the 'genesis timestamp' as input then, right?
I'm OK with either approach honestly.

Reply:
I kinda like having the proofs be their own thing, we already have a few such functions in the EP